README.md

```diff
@@ -15,16 +15,28 @@ This tool generate tree graph from ARF xml report from OpenSCAP scan.
   * [inquirer](https://pypi.org/project/inquirer/)
 
 ### Installation
-
+#### Fedora 30 and later
+```bash
+dnf install oval-graph
+# Install inquirer, if you want have nice cli futures. (optional)
+pip3 install inquirer
+```
+#### Install via [pip](https://pypi.org/project/oval-graph/)
+```bash
+pip3 install oval-graph
```

Comment:
All those install commands require root, so `sudo` would be appropriate. You can add that if you install using `pip3 install --user oval-graph`, binaries will end up in `$HOME/.local/bin`, but it won't require root privileges.

Reply:
ditto - `sudo` is missing.